process/block/postprocess/basePostProcess.go

```diff
@@ -130,3 +131,10 @@ func (bpp *basePostProcessor) verifyMiniBlock(createMBs map[uint32]*block.MiniBl
 
 	return nil
 }
+
+func (bpp *basePostProcessor) GetCreatedInShardMiniBlock() *block.MiniBlock {
```

Comment:
Please correct the log on line 57 UnsignedTransactionUnit.Put -> something else involving bpp.storageType

Reply:
done.